salt/engines/logstash.py

```diff
@@ -40,13 +41,19 @@ def __virtual__():
 log = logging.getLogger(__name__)
 
 
-def start(host, port=5959, tag='salt/engine/logstash'):
+def start(host, port=5959, proto='tcp', tag='salt/engine/logstash'):
```

Comment:
Can you move the new `proto` kwarg to the end of the list so we don't have an API change here? Otherwise I think this looks good.